test/specs/mainnet/SendFlow/Smoke_NativeTokenTransaction.spec.js

```diff
@@ -4,8 +4,13 @@ dotenv.config(); // init dotenv
 import { assert } from "chai";
```

Comment:
Did you upstream something here? This wasn't part of this PR originally

Reply:
No, I have not upstreamed here.